cmd/preview/deploy.go

```diff
@@ -95,15 +95,21 @@ func Deploy(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			previewURL, err := getPreviewURL(ctx, name)
```

Comment:
same here, we could store the preview object returned by the `deployPreview` api call.